src/lib.rs

```diff
@@ -126,8 +126,17 @@ pub use xor_name;
 
 /// The minimum size (before compression) of data to be self-encrypted, defined as 3B.
 pub const MIN_ENCRYPTABLE_BYTES: usize = 3 * MIN_CHUNK_SIZE;
-/// The maximum size (before compression) of an individual chunk of a file, defined as 500kiB.
-pub const MAX_CHUNK_SIZE: usize = 512 * 1024;
+/// The default maximum size (before compression) of an individual chunk of a file, defaulting as 1MiB.
+const DEFAULT_MAX_CHUNK_SIZE: usize = 1024 * 1024;
+
+lazy_static! {
+    /// The maximum size (before compression) of an individual chunk of a file, defaulting as 1MiB.
+    pub static ref MAX_CHUNK_SIZE: usize = std::option_env!("MAX_CHUNK_SIZE")
+        .unwrap_or("1048576")
+        .parse::<usize>()
```

Comment:
What is the rationale behind trying to parse this string here? Can we not just drop these lines?